frontend/public/storage/components/storage-overview/storage-overview.jsx

```diff
@@ -19,6 +19,8 @@ import { coFetchJSON } from '../../../co-fetch';
 const REFRESH_TIMEOUT = 5000;
 
 const CEPH_STATUS = 'ceph_health_status';
+const CEPH_PG_ACTIVE = 'ceph_pg_active';
```

Comment:
can you add a `QUERY` suffix to these variables?

Reply:
done